ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.19.yaml

```diff
@@ -450,33 +459,36 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-managed-identity
-- as: azure-ipi-oidc-managed-identity-user-defined-f14
-  cron: 29 15 8,22 * *
+- as: azure-ipi-oidc-managed-identity-user-defined-mini-perm-f14
+  cron: 31 21 12,28 * *
   steps:
     cluster_profile: azure-qe
     env:
+      AZURE_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
       AZURE_MANAGED_IDENTITY_TYPE: user-defined
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       EXTRACT_MANIFEST_INCLUDED: "true"
```

Comment:
do we need "ENABLE_MIN_PERMISSION_FOR_STS" for this one?

Reply:
I think enable `ENABLE_MIN_PERMISSION_FOR_STS` would be better as `-mini-perm-` job mean enabled mini perm test on both installer and ccoctl.

Reply:
Discussed with @jianlinliu before for this issue.

When setting `ENABLE_MIN_PERMISSION_FOR_STS` to "true",  step `azure-provision-service-principal-minimal-permission` must be included into this workflow as well to create two service principal for both installer and ccoctl.
Here is spacial case which does not need to create service principal, if step `azure-provision-service-principal-minimal-permission` is added, one useless service principal will be created. Maybe we can introduce new ENV to control whether or not to create SP, but this make the script a little complicated.

So keep current logical in this workflow, and this job is only run in verification test, I guess it should be acceptable.

Reply:
Yeah, this is a special case, the main test point is to run installer on managed-identity hosts, but for such cases, it only support manual cco, so enable oidc here.

Reply:
ok, get it.